client/src/components/History/Content/Collection/CollectionProgress.test.ts

```diff
@@ -0,0 +1,74 @@
+import { getLocalVue } from "@tests/jest/helpers";
+import { mount } from "@vue/test-utils";
+
+import { JobStateSummary } from "./JobStateSummary";
+
+import CollectionProgress from "./CollectionProgress.vue";
+
+const localVue = getLocalVue();
+
+async function mountComponent(dsc: object) {
+    const jobStateSummary = new JobStateSummary(dsc);
+
+    const wrapper = mount(CollectionProgress as object, {
+        propsData: {
+            summary: jobStateSummary,
+        },
+        localVue,
+    });
+
+    return wrapper;
```

Comment:
```suggestion
    });

    await flushPromises();

    return wrapper;
```

I would suggest putting this here so you don't have to remember to call `await wrapper.vm.$nextTick();` after each `mountComponent` call.